### ISSUE TITLE
error - TypeError: value.split is not a function – when adjusting brightness with Wemo link

### ISSUE THREAD

**Issue by benfysh** [bwp91/homebridge-platform-wemo#210]
When adjusting the brightness for my bulbs via the Wemo link I get the following error in the logs the bulb adjust as expected and on and off work fine without any log entries.
...
**Details of your setup.**
Homebridge v1.2.3 on a raspberry pi 4
...
Which version of this plugin (homebridge-platform-wemo) do you have?
homebridge-platform-wemo v2.0.0
...
```
[10/26/2020, 18:45:18] [WeMo Platform] [Dining room] setting brightness to [28%].
[10/26/2020, 18:45:19] [WeMo Platform] [Dining room] updating brightness to [68%] error - TypeError: value.split is not a function
    at deviceLink.externalBrightnessUpdate (/usr/lib/node_modules/homebridge-platform-wemo/lib/device/link.js:168:32)
    at module.exports.<anonymous> (/usr/lib/node_modules/homebridge-platform-wemo/lib/device/link.js:60:16)
    at module.exports.emit (events.js:326:22)
    at /usr/lib/node_modules/homebridge-platform-wemo/lib/wemo-client.js:384:16
    at Parser.<anonymous> (/usr/lib/node_modules/homebridge-platform-wemo/node_modules/xml2js/lib/parser.js:304:18)
    at Parser.emit (events.js:314:20)
    at SAXParser.onclosetag (/usr/lib/node_modules/homebridge-platform-wemo/node_modules/xml2js/lib/parser.js:262:26)
    at emit (/usr/lib/node_modules/homebridge-platform-wemo/node_modules/sax/lib/sax.js:624:35)
    at emitNode (/usr/lib/node_modules/homebridge-platform-wemo/node_modules/sax/lib/sax.js:629:5)
    at closeTag (/usr/lib/node_modules/homebridge-platform-wemo/node_modules/sax/lib/sax.js:889:7)
    at SAXParser.write (/usr/lib/node_modules/homebridge-platform-wemo/node_modules/sax/lib/sax.js:1436:13)
    at Parser.exports.Parser.Parser.parseString (/usr/lib/node_modules/homebridge-platform-wemo/node_modules/xml2js/lib/parser.js:323:31)
    at Parser.parseString (/usr/lib/node_modules/homebridge-platform-wemo/node_modules/xml2js/lib/parser.js:5:59)
    at Object.exports.parseString (/usr/lib/node_modules/homebridge-platform-wemo/node_modules/xml2js/lib/parser.js:369:19)
    at Object.StatusChange (/usr/lib/node_modules/homebridge-platform-wemo/lib/wemo-client.js:382:14)
    at /usr/lib/node_modules/homebridge-platform-wemo/lib/wemo-client.js:419:22
[10/26/2020, 18:45:19] [WeMo Platform] [Dining room] setting brightness to [27%].

```
...

**Comment by bwp91** [comment]
Hi, thanks for this.
Please could you run this in the homebridge terminal:
`sudo npm i -g homebridge-platform-wemo@beta`
To see if this fixes it?

**Comment by bwp91** [comment]
This should be fixed in v2.1.0!

**Comment by benfysh** [comment]
> This should be fixed in v2.1.0!

Confirmed, thanks for the quick turnaround :)
